src/pages/SamplePage/SamplePage.tsx

```diff
@@ -7,8 +7,8 @@ import AppBanner from '../../AppBanner';
 const SampleComponent = lazy(() => import('../../components/SampleComponent/SampleComponent'));
 
 import './SamplePage.scss';
-import '../../App.scss';
-import '../../shared/style.scss';
+// import '../../App.scss';
+// import '../../shared/style.scss';
 
```

Comment:
Uncomment or remove these if not needed.

Reply:
Done.